src/pkg/packager/create.go

```diff
@@ -385,15 +398,16 @@ func (p *Packager) addComponent(component types.ZarfComponent) (*types.Component
 		spinner := message.NewProgressSpinner("Loading %d K8s manifests", manifestCount)
 		defer spinner.Success()
 
-		if err := utils.CreateDirectory(componentPath.Manifests, 0700); err != nil {
-			return nil, fmt.Errorf("unable to create manifest directory %s: %w", componentPath.Manifests, err)
-		}
-
 		// Iterate over all manifests.
 		for _, manifest := range component.Manifests {
 			for idx, f := range manifest.Files {
 				// Copy manifests without any processing.
 				spinner.Updatef("Copying manifest %s", f)
+				if utils.IsURL(f) {
+					dst := filepath.Join(componentPath.Temp, fmt.Sprintf("manifest-%d", idx))
+					utils.DownloadToFile(f, dst, component.CosignKeyPath)
```

Comment:
This should just download to the correct manifest location instead of downloading to `temp` and then copying.  `temp` isn't guaranteed to exist here either and will break if not.